sections/semantics-sections.include

```diff
@@ -142,102 +142,98 @@
-  <{article}>, but it is technically redundant in that case (since it's self-evident that
-  the page is a single composition, as it is a single document).
-
+  
   <div class="example">
-    This example shows a blog post using the <{article}> element, with some schema.org
-    annotations:
-
-    <pre highlight="html">
-      &lt;article typeof="schema:BlogPosting"&gt;
+ <p>The following is an example of a blog post extract, marked up using the <code>article</code> element:</p>
+ <pre highlight="html">
+ &lt;article&gt;
+  &lt;header&gt;
+   &lt;h2&gt;&lt;a href="http://herbert.io"&gt;Short note on wearing shorts&lt;/a&gt;&lt;/h2&gt;
+    &lt;p&gt;Posted on Wednesday, 10 February 2016 by Patrick Lauke.
+    &lt;a href="http://herbert.io/short-note/#comments"&gt;6 comments&lt;/a&gt;&lt;/p&gt;
+  &lt;/header&gt;
+  &lt;p&gt;A fellow traveller posed an interesting question: Why do you wear shorts rather than 
+  longs? The person was wearing <em>culottes</em> as the time, so I considered the question equivocal in nature, 
+  but I attempted to provide an honest answer despite the dubiousness of the questioner's dress.&lt;/p&gt; 
+  &lt;p&gt;The short answer is that I enjoy wearing shorts, the long answer is...&lt;/p&gt;
+  &lt;p&gt;&lt;a href="http://herbert.io/short-note/"&gt;Continue reading: Short note on 
+  wearing shorts&lt;/a&gt;&lt;/p&gt;
+ &lt;/article&gt;</pre>
+ </div>
+
+ <p class="note">The schema.org  vocabulary can be used to provide more granular information about 
+  the type of article, using the <a href="http://schema.org/Article">CreativeWork - Article</a> 
+  subtypes, other information such as the publication date for the article can also be provided.</p>
+  
+  <div class="example">
+  <p>This example shows a blog post using the <code>article</code> element, with some schema.org
+  annotations: </p>
+  <pre highlight="html">      
+      &lt;article itemscope <mark>itemtype="http://schema.org/BlogPosting"</mark>&gt;
         &lt;header&gt;
-          &lt;h2 property="schema:headline"&gt;The Very First Rule of Life&lt;/h2&gt;
-          &lt;p&gt;&lt;time property="schema:datePublished" datetime="2009-10-09"&gt;3 days ago&lt;/time&gt;&lt;/p&gt;
-          &lt;link property="schema:url" href="?comments=0"&gt;
+          &lt;h2 itemprop="headline"&gt;The Very First Rule of Life&lt;/h2&gt;
+          &lt;p&gt;&lt;time <mark>itemprop="datePublished"</mark> datetime="2016-02-28"&gt;3 days ago&lt;/time&gt;&lt;/p&gt;
         &lt;/header&gt;
         &lt;p&gt;If there's a microphone anywhere near you, assume it's hot and
         sending whatever you're saying to the world. Seriously.&lt;/p&gt;
         &lt;p&gt;<em>...</em>&lt;/p&gt;
         &lt;footer&gt;
-          &lt;a property="schema:discussionUrl" href="?comments=1"&gt;Show comments...&lt;/a&gt;
+          &lt;a itemprop="discussionUrl" href="?comments=1"&gt;Show comments...&lt;/a&gt;
         &lt;/footer&gt;
```

Comment:
(And elsewhere)

((As Yandex' representative for schema.org))

please use RDFa for schema.org instead of microdata. While Schema is format-agnostic, microdata has technical  problems the other formats don't, and isn't a Recommendation.